src/main/resources/application.yaml

```diff
@@ -37,18 +40,24 @@ operations-app:
     batch-summary: "/api/v1/batch"
     batch-transaction: "/api/v1/batch/transactions"
 
+mock-payment-schema:
+  contactpoint: "http://localhost:8080"
+  endpoints:
+    authorization: "/batches/"
+
 channel:
   hostname: "http://ph-ee-connector-channel"
 
 cloud:
   aws:
     enabled: true
-    s3-base-url: "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com"
+    s3-base-url: "https://fynarfin-bucket.s3.ap-south-1.amazonaws.com"
```

Comment:
Please rebase, S3 changes must be already available in upstream

Reply:
@fynmanoj As mentioned in previous review comments. I do not see these changes in master. Please confirm if the changes were merged.

Reply:
These changes are updated in the chart to work with SIT, for local development you can change them, but no need to push these to code base